src/VisualStudio/IntegrationTest/TestUtilities/InProcess/TextViewWindow_InProc.cs

```diff
@@ -329,24 +329,28 @@ private async Task<IEnumerable<ISuggestedAction>> GetLightBulbActionsAsync(ILigh
             return await SelectActionsAsync(actionSets);
         }
 
-        public void ApplyLightBulbAction(string actionName, FixAllScope? fixAllScope, bool blockUntilComplete)
+        public bool ApplyLightBulbAction(string actionName, FixAllScope? fixAllScope, bool blockUntilComplete)
         {
             var lightBulbAction = GetLightBulbApplicationAction(actionName, fixAllScope, blockUntilComplete);
             var task = ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
                 var activeTextView = GetActiveTextView();
-                await lightBulbAction(activeTextView);
+                return await lightBulbAction(activeTextView);
             });
 
             if (blockUntilComplete)
             {
-                task.Join();
+                var result = task.Join();
+                DismissLightBulbSession();
```

Comment:
why this change?

Reply:
Code fix suggestions are not recomputed while the light bulb is visible. Either way, after applying a code fix, we need to dismiss the session to match the behavior you get when a user applies a code fix by interacting with the menus.

Reply:
ok.  so this was a bug, and we shoudl have done this regardless of tracking if things were applied or not?